studio/components/interfaces/TableGridEditor/SidePanelEditor/TableEditor/Column.tsx

```diff
@@ -59,7 +58,7 @@ const Column: FC<Props> = ({
   onEditRelation = () => {},
```

Comment:
we should change all these to noops (from lodash) too
```suggestion
  onEditRelation = noop,
```

Reply:
Fixed.